components/Document/api/fetchPostFromS3.ts

```diff
@@ -14,7 +15,29 @@ const fetchPostFromS3 = async ({ s3Url, cleanIntroEmptyContent = true }: Props):
         msg: `Error fetching post from S3`,
       });
     }
-    let _html = await response.text();
+    let _html = await response.text().then((text) =>
+      sanitizeHtml(text, {
```

Comment:
Not sure if we need to sanitize these, but in theory, more than just the following three tags can exist as part of the response. For example, `table`, `figure`, ...

Reply:
Those are included in the default allowed tags which I'm concatenating with some missing tags https://www.npmjs.com/package/sanitize-html

Reply:
Maybe I should be more explicit and just define all of them here?

Reply:
Ah gotcha. I think if you add a comment, it should be more than sufficient